src/broadcast/listenbrainzlistener/listenbrainzservice.cpp

```diff
@@ -0,0 +1,67 @@
+#include "broadcast/listenbrainzlistener/listenbrainzservice.h"
+
+#include <QJsonObject>
+
+#include "broadcast/listenbrainzlistener/listenbrainzjsonfactory.h"
+#include "broadcast/listenbrainzlistener/networkmanager.h"
+#include "moc_listenbrainzservice.cpp"
+#include "preferences/listenbrainzsettings.h"
+
+ListenBrainzService::ListenBrainzService(UserSettingsPointer pSettings)
+        : m_request(ListenBrainzAPIURL),
+          m_latestSettings(ListenBrainzSettingsManager::getPersistedSettings(pSettings)),
+          m_COSettingsChanged(ListenBrainzSettingsManager::kListenBrainzSettingsChanged) {
+    connect(&m_manager,
+            &QNetworkAccessManager::finished,
+            this,
+            &ListenBrainzService::slotAPICallFinished);
+    connect(&m_COSettingsChanged,
+            &ControlPushButton::valueChanged,
+            this,
+            &ListenBrainzService::slotSettingsChanged);
+    m_request.setHeader(QNetworkRequest::ContentTypeHeader, "application/json");
+    if (m_latestSettings.enabled) {
+        m_request.setRawHeader("Authorization", "Token " + m_latestSettings.userToken.toUtf8());
+    }
+}
+
+void ListenBrainzService::slotBroadcastCurrentTrack(TrackPointer pTrack) {
+    Q_UNUSED(pTrack);
+    /*if (!pTrack || !m_latestSettings.enabled)
```

Comment:
What is this commented code? Does it work? Are you going to implement this or add a TODO comment?